src/components/LanguageSwitcher/LanguageSwitcher.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "gatsby";
+
+import cs from "../../images/cs.png";
+import en from "../../images/en.png";
+
+import languages from "../../i18n/languages";
+import getLocalizedPath from "../../i18n/getLocalizedPath";
```

Comment:
It would be better to import both from `../../i18n` as I described below..

Reply:
:+1: